src/NuGet.Core/NuGet.Configuration/Settings/Items/AuthorItem.cs

```diff
@@ -0,0 +1,103 @@
+// Copyright(c) .NET Foundation.All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Xml.Linq;
+using NuGet.Shared;
+
+namespace NuGet.Configuration
+{
+    public sealed class AuthorItem : TrustedSignerItem, IEquatable<AuthorItem>
+    {
+        public override string ElementName => ConfigurationConstants.Author;
+
+        protected override HashSet<string> RequiredAttributes => new HashSet<string>() { ConfigurationConstants.NameAttribute };
+
+        public AuthorItem(string name, params CertificateItem[] certificates)
+            : base(name, certificates)
+        {
+        }
+
+        internal AuthorItem(XElement element, SettingsFile origin)
+            : base(element, origin)
+        {
+        }
+
+        internal override SettingBase Clone()
+        {
+            var newItem = new AuthorItem(Name, Certificates.ToArray());
```

Comment:
What's the intention of the cloning here? 
This is not a deep clone as Certificates itself is mutable.

Reply:
You are right, certificates should also be cloned. This is to follow the pattern of all `SettingItem`. Cloning is used when getting the values to make sure the user gets copies instead of the values that hold the reference to the XElements